Gruntfile.js

```diff
@@ -0,0 +1,137 @@
+module.exports = function(grunt) {
+  "use strict";
+
+  var srcDir = grunt.option('source-path') || 'web/app/themes';
```

Comment:
you know how the path to this file changes to 'bedrock/web/app/themes' once it's moved by the Dockerfile ... will any of this change in this refactor?

Reply:
No.  That's for later.